tee-worker/cli/src/trusted_operation.rs

```diff
@@ -369,7 +363,7 @@ fn send_direct_request<T: Decode + Debug>(
 fn send_direct_vc_request<T: Decode + Debug>(
 	cli: &Cli,
 	trusted_args: &TrustedCli,
-	operation_call: &TrustedOperation<TrustedCallSigned, Getter>,
+	operation_call: &TrustedVCRequestSigned,
```

Comment:
The same argument: this will limit the interface only for `VC request`. Should we do it?